ports/windows/Makefile

```diff
@@ -92,6 +92,12 @@ include $(TOP)/py/mkrules.mk
 
 RUN_TESTS_SKIP += -e math_fun -e float2int_double -e float_parse -e math_domain_special
 
+ifeq ($(MSYSTEM),MINGW32)
+# compliler optimizations break float formatting
+# https://github.com/micropython/micropython/pull/10267
+RUN_TESTS_SKIP += -e format_ints_doubleprec -e format_modulo2_intbig
```

Comment:
Is there a reason to do the skipping here instead of directly in run_tests.py?

Reply:
Merging #10267 will make this change here go away.